packages/v4/patternfly-docs/content/design-guidelines/components/form/forms.md

```diff
@@ -147,17 +149,17 @@ Example: For a form with multiple field groups, distinguish between each one by
 
 To view an interactive example of field groups, see our [field groups demo](/components/form#field-groups).
 
-## User guidance
+## Usage
 
 Due to their interactive nature, forms often include features that help communicate field importance and guidance as users complete them. 
 
+### Types of user guidance
 Types of user guidance include:
+* [**Required fields:**](#Required-fields) Call attention to input fields crucial to user success.
```

Comment:
Same here about needing to make them sentence case

so 
#required-fields
#user-help 
etc